notebooks/parsing_mbox.ipynb

```diff
@@ -40,7 +40,7 @@
   },
```

Comment:
<p>You could add an "emails.head()" at the bottom here so that others can see the example of the dataframe you've created</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>